cedar-policy/CHANGELOG.md

```diff
@@ -79,6 +78,19 @@ Cedar Language Version: 4.0
 - (*) JSON format Cedar policies will now fail to parse if the action scope
   constraint contains a non-action entity type, matching the behavior for
   human-readable Cedar policies. (#943, resolving #925)
+
+## [3.2.4] - 2024-08-07
+Cedar Language Version: 3.3
+
+_Note:_ 3.2.2 and 3.2.3 skipped to maintain consistency with the `cedar-wasm` package
+
+### Changed
+
+- (*) `unknown()` is no longer a valid extension function if `partial-eval`
+  is not enabled as a feature. (#1101, resolving #1096)
+
```

Comment:
I'd argue this is `Fixed` rather than `Changed`, I think.  If not, it's technically a breaking change, kinda

Reply:
Yeah, that's fair. I also think this was a bug fix.